README.md

```diff
@@ -6,8 +6,8 @@
 # SSV Node
 
 [![API Reference](https://img.shields.io/badge/API%20Reference-blue)](https://pkg.go.dev/github.com/ssvlabs/ssv)
-![Github Actions](https://github.com/ssvlabs/ssv/actions/workflows/unit-test.yml/badge.svg?branch=main)
-![Github Actions](https://github.com/ssvlabs/ssv/actions/workflows/lint.yml/badge.svg?branch=main)
+[![Github Actions](https://github.com/ssvlabs/ssv/actions/workflows/unit-test.yml/badge.svg?branch=main)](https://github.com/ssvlabs/ssv/actions/workflows/unit-test.yml)
```

Comment:
The link shows unit test workflows for all PRs/branches, including the failing ones. Since the badge is expected to show the workflow is passing, should we filter only pushes to `main`?

```suggestion
[![Github Actions](https://github.com/ssvlabs/ssv/actions/workflows/unit-test.yml/badge.svg?branch=main)](https://github.com/ssvlabs/ssv/actions/workflows/unit-test.yml?query=branch%3Amain+event%3Apush)
```